packages/ui/src/icons/icon/icon.native.tsx

```diff
@@ -16,7 +16,7 @@ export const Icon = forwardRef<Component, IconProps>(({ children, ...props }, re
   return (
     <>
       {React.cloneElement(child, {
-        color: theme.colors['ink.action-primary-default'],
+        color: props.color ?? theme.colors['ink.action-primary-default'],
```

Comment:
Is this change nec? We have always been able to override the icon color w/out this. This should just set the default color?

Reply:
It does make the fallback color more explicit and thus easier to debug. 

Of course it can always be overriden without this, but the underlying `svg` usually has something like `stroke="currentColor"` and it's not obvious to me what that default is - having something explicitly set rather than a side effect seems like an improvement.

Reply:
I would have to find the resource I used when implementing this with `[react-native-svg-transformer](https://github.com/kristerkari/react-native-svg-transformer)`. I know it was the suggested way to set the svg `currentColor` using a theme color. I'm not totally following your explanation tbh.

Reply:
> Is this change nec? We have always been able to override the icon color w/out this. This should just set the default color?

I think I misunderstood your original comment - I thought it was regarding `theme.colors['ink.action-primary-default']` which had already been there, you're right this isn't necessary given that the props are spread as the final argument.